content/en/agent/proxy.md

```diff
@@ -134,20 +134,114 @@ proxy_password: my_password
 
 Do not forget to [restart the Agent][1] for the new settings to take effect.
 
+## Squid
 
+[Squid][2] is a caching proxy for the web supporting HTTP, HTTPS, FTP, and more. It runs on most available operating systems, including Windows, and is licensed under the GNU GPL. Squid is easy to configure and widely used for proxying traffic in a variety of different situations. 
+
+This is the best option if you do not have a web proxy readily available in your network and you wish to proxy a large number of Agents. 
+
+### Proxy forwarding with Squid
+
+#### Squid configuration
+
+Squid should be installed on a host that has connectivity to Datadog. Use your operating system's package manager, or install the software directly from [Squid's project page][2].
+
+Most of the settings below are configured within Squid's configuration file, usually located at `/etc/squid/squid.conf` on Linux or `C:\squid\etc\squid.conf` in Windows. 
+
+##### Configure Squid to only send traffic to Datadog
+
+First, configure Squid to accept traffic from the correct internal IP address ranges where your Agents are running. Many of these ACLs may already be defined.
+
+Ensure that your `squid.conf` file contains the following settings. The IP ranges listed below may be different for your environment depending on your network.
+
+```conf
+http_port 0.0.0.0:3128
+
+acl local src 127.0.0.1/32
+
+acl Datadog dstdomain .{{< region-param key="dd_site" >}}
+
+http_access allow Datadog
+http_access allow local manager
+```
+
+##### Start Squid
+
+Restart Squid so that your configurations can be applied. If you're configuring Squid on Windows, you will first need [to configure Squid as a system service][9]
```

Comment:
```suggestion
Restart Squid so that your configurations can be applied. If you're configuring Squid on Windows, you will first need [to configure Squid as a system service][9].
```